xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/table/livetabledate.js

```diff
@@ -105,10 +108,22 @@
           }
         });
 
+        input.on('showCalendar.daterangepicker', function(event) {
+          event.target.next('.daterangepicker').setAttribute('style',
+            'display:block; width: max-content;');
```

Comment:
I wonder if this style shouldn't be better placed in some CSS file (but I haven't checked if there is a suitable one including a suitable selector).

Reply:
Blocks move a lot here and are generated using bootstrap 3.
If I remember well, I tried just adding a class with style attached to it, but the style would get taken over by bootstrap generating a 'style' attribute inline over it.
This is the only way I found to change this couple of properties from the bootstrap default (that needed to be overridden for accessibility) on this element.
The other style properties I could just do it with regular selectors: see [xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/table/livetable.css](https://github.com/xwiki/xwiki-platform/pull/2149/files#diff-8eaa7402bfc75704d30fcf9128bf89987aeff1cc3a38eb20e66ea83ed6b84c53).

____

The time I spent on understanding this and the bad looks of this code is also why I decided to not update the whole bootstrap modal for accessibility. It'd probably be faster coding it from scratch again :D
As of now, Assistive Techs (AT) users have a proper way to select date ranges, even if it's not as nice as the one regular users get.